src/Effectful/Log.hs

```diff
@@ -64,6 +66,13 @@ runLog component logger maxLogLevel = reinterpret reader $ \env -> \case
       , leData = []
       , leMaxLogLevel = maxLogLevel
       }
+    logException :: (IOE :> es, Log :> es) => SomeException -> Eff es a
+    logException (SomeException e) = do
+      time <- liftIO getCurrentTime
+      logEnv <- getLoggerEnv
+      liftIO $
+        logMessageIO logEnv time LogAttention "Uncaught exception" $ object ["error" .= (pack . show $ e)]
```

Comment:
You can use logAttention here (and you don't need pack, same as in the log-base PR).